src/renderer/src/components/common/Flex/Row.tsx

```
@@ -0,0 +1,16 @@
+import { styled } from 'styled-components'
+import type { FlexProps } from './Flex.type'
+
+const Row = ({ children, gap, justifyContent, alignItems, width, height, style }: FlexProps) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)